tests/ui/pages/desktop/details.py

```diff
@@ -0,0 +1,31 @@
+from pypom import Region
+from selenium.webdriver.common.by import By
+
+from base import Base
+
+
+class Details(Base):
+    """ Details page. """
+
+    _wait_element = (By.CLASS_NAME, 'addon-details')
+
+    def wait_for_page_to_load(self):
+        self.wait.until(lambda _: self.find_element(
+            *self._wait_element).is_displayed())
```

Comment:
I'm not a fan of `_wait_element`.. just add an `is_displayed` to `DescriptionHeader`.